osu.Game.Resources/Localisation/Web/ChatStrings.cs

```diff
@@ -137,7 +137,7 @@ public static class ChatStrings
         /// <summary>
         /// "type message..."
         /// </summary>
-        public static LocalisableString InputPlaceholder => new TranslatableString(getKey(@"input.placeholder"), @"type message...");
+        public static LocalisableString InputPlaceholder => new TranslatableString(getKey(@"input.placeholder"), @"press enter to type message...");
```

Comment:
Are you sure about this?

![2024-08-04 23 13 57@2x](https://github.com/user-attachments/assets/f5985856-1ea1-4788-b777-407668dd6ab1)

Reply:
This is at the suggestion of @frenzibyte in [osu! issue #292264](https://github.com/ppy/osu/issues/29264)

From what I can tell though, the chatbox is always in focus and taking input, so the "enter" hint may not be necessary

Reply:
The screenshot I'm showing is in the chat overlay, where "enter" is not required. You're changing the text in all usages, which would break this one.

Reply:
Aah I see - thank you for the clarification. I will close this and attempt to re-implement for just the multiplayer in-game chat.